packages/playwright-test/src/reporters/raw.ts

```diff
@@ -71,7 +71,7 @@ export type JsonTestCase = {
 
 export type JsonAttachment = {
   name: string;
-  body?: string;
+  body?: string | Buffer;
```

Comment:
Not quite "json" attachment anymore.

Reply:
Agreed, but it is the right thing to do here...